infrastructure/iac/template/index.html

```diff
@@ -95,5 +95,10 @@ <h2>References</h2>
   {{end}}
 </div>
 </body>
-${ideScript}
+<script nonce="{{.Nonce}}">
+  (function(){
```

Comment:
this immediately invoked function is removed from intellij code and added here so we can add scripts from different sources in this same function that's loaded on startup.